homeassistant/components/reolink/strings.json

```
@@ -31,7 +31,7 @@
       "cannot_connect": "Failed to connect, check the IP address of the camera",
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "not_admin": "User needs to be admin, user \"{username}\" has authorization level \"{userlevel}\"",
-      "password_incompatible": "Password contains incompatible special character or is too long, maximum 31 characters and only these characters are allowed: a-z, A-Z, 0-9 or {special_chars}",
+      "password_incompatible": "Password contains incompatible special character or is too long, maximum 31 characters and only these characters are allowed: a-z, A-Z, 0-9 or {special_chars}. The streaming protocols necessitate these additional password restrictions.",
```

Let's use a bit more simple words.
```suggestion
      "password_incompatible": "Password contains incompatible special character or is too long. Passwords should be a maximum of 31 characters and only the following characters are allowed: a-z, A-Z, 0-9 or {special_chars}. These are password restrictions by Reolink.",
```

---

But the whole point is that these password requirements are more strict than the Reolink password requirements.

For example "#" is allowed by reolink, but will cause issues in the RTSP/RTMP stream (confirmed by reolink firmware engineers). The reolink app does not use RTSP/RTMP.

---

I guess this is fine.